ansible/playbooks/adhoc/uninstall.yml

```diff
@@ -33,21 +47,46 @@
         - kube-scheduler
         - kube-addons
         - etcd
+        - etcd2
       failed_when: false
 
     - name: Remove packages
       action: "{{ ansible_pkg_mgr }} name={{ item }} state=absent"
-      when: not is_atomic | bool
+      when: not is_coreos and not is_atomic | bool
       with_items:
         - flannel
         - kubernetes-node
         - kubernetes-master
         - etcd
+        - etcd2
```

Comment:
@adamschaub I don;t think we need to remove etcd2 since this is not is_coreos. Feel free to send a follow-on PR that removes if the change works in your lab.